boards/arm/lpcxpresso54114_m0/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,61 @@
+.. _lpcxpresso54114_m0:
+
+NXP LPCXPRESSO54114
+#####################
+
+Overview
+********
+
+See :ref:`lpcxpresso54114` for a general overview of the LPCXpresso54114 board.
+The Cortex-M0+ is a secondary core on the board's SoC.
+
+Programming and Debugging
+*************************
+
+The LPCXpresso54114 includes the LPC-Link2 serial and debug adapter built into
+the board to provide debugging, flash programming, and serial communication
+over USB. LPC-Link2 can be configured with Segger J-Link or CMSIS-DAP firmware
+variants to support corresponding debug tools. Currently only the Segger J-Link
+tools are supported for this board in Zephyr, therefore you should use the
+Segger J-Link firmware variant.
+
+Before you start using Zephyr on the LPCXpresso54114, download and run
+`LPCScrypt`_ to update the LPC-Link2 firmware to the latest version, currently
+``Firmware_JLink_LPC-Link2_20160923.bin``. Serial communication problems, such
+as dropping characters, have been observed with older versions of the firmware.
+
+The code for the secondary core is linked into the primary core binary file.
+Startup code copies the secondary core's code into an appropriate location
```

Comment:
Might be work clarifying (assuming I've got this right):

    Startup code for the primary core copies the secondary core's code into an
    appropriate location in RAM and starts the secondary core's execution.